rulesets/standard.json

```diff
@@ -57,6 +57,14 @@
             ]
         }
     },
+    "//planned-disbursement/provider-org | //planned-disbursement/receiver-org | //transaction/provider-org | //transaction/receiver-org | //recipient-org-budget/recipient-org": {
+        "regex_matches": {
+            "cases": [
+                { "regex": "^(?!\s*$).+",
+                  "paths": [ "narrative" ], "condition": "count(@ref)==1" }
```

Comment:
Wonder if we should look for `narrative/text()` here?

Reply:
Because it's an element, this is automatically appended.

Reply:
https://github.com/IATI/IATI-Rulesets/blob/version-2.02/iatirulesets/__init__.py#L11

Reply:
Sounds good - as we found, this doesn't appear in the [ruleset spec](http://iatistandard.org/202/rulesets/ruleset-spec) so would be oblivious to a ruleset user.  I'll make a Monday project to try to improve the writeup of the spec at some point!